lucien-ios/LucienConstants.swift

```diff
@@ -34,6 +39,18 @@ final class LucienConstants {
     static let coverButtonOpacity = CGFloat(0.6)
     static let buttonBorderRadius = CGFloat(10)
 
+    // MARK: - CoverPhotoButton Default Button Constants
+
+    static let coverPhotoDefaultAlpha =  CGFloat(1.0)
+    static let coverPhotoDefaultShadowOpacity =  Float(0.0)
+    static let coverPhotoDefaultShadowRadius =  CGFloat(0.0)
+    static let coverPhotoBottomImageInset =  CGFloat(70)
+    static let coverPhotoNormalStateAttributedTitle = NSAttributedString(string: "Take Cover Photo", attributes: [NSAttributedStringKey.font: LucienTheme.Fonts.muliBold(size: 16.0) ?? UIFont(), NSAttributedStringKey.foregroundColor: UIColor.white])
+    static let coverPhotoHighlightedStateAttributedTitle = NSAttributedString(string: "Take Cover Photo", attributes: [NSAttributedStringKey.font: LucienTheme.Fonts.muliBold(size: 16.0) ?? UIFont(), NSAttributedStringKey.foregroundColor: UIColor.white.withAlphaComponent(0.5)])
+    static let coverPhotoTopTitleEdgeInset =  CGFloat(25)
+
+
+
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Vertical Whitespace Violation: Limit vertical whitespace to a single empty line. Currently 3. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.intrepid.io/coding_rules#rule_key=SwiftLint%3Avertical_whitespace)